cdap-validation/bin/baseref_configs

```diff
@@ -0,0 +1,5 @@
+cdap:cdap_master_heapsize='1024m':bytes:exact
+hbase:hbase_master_heapsize='4096m-':bytes:range
+yarn:nodemanager_heapsize='512m-99999m':bytes:range
+hbase:hbase.security.authorization='false':alpha:exact
+yarn:yarn.acl.enable='true':alpha:exact
```

Comment:
There's nothing to inform a user what any of this means. At the very least, there should be a comment header. Best case would be some structure to the file to make it easier to parse. A format such as JSON will be simpler to handle than your own format, simply for validation reasons. For example, what if I have a value of `3:1` with alpha? How do you handle this? The rules for known formats are specific and "universally" known.